src/test/java/com/glovoapp/planningpoker/e2e/spectatorbox/SpecSelectedTest.java

```diff
@@ -0,0 +1,47 @@
+package com.glovoapp.planningpoker.e2e.spectatorbox;
+
+import com.glovoapp.planningpoker.e2e.WebDriverFactory;
+import com.glovoapp.planningpoker.e2e.pages.MainPage;
+import com.glovoapp.planningpoker.e2e.pages.SessionPage;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.Optional;
+
+import static com.glovoapp.planningpoker.e2e.WebDriverFactory.getDriver;
+
+class SpecSelectedTest {
+
+    private final WebDriver driver = getDriver();
+    private MainPage mainPage = new MainPage(driver);
+    private SessionPage sessionPage = new SessionPage(driver);
+    private final String userName = "username";
+
+    @AfterEach
+    void releaseDriver() {
+        WebDriverFactory.releaseDriver(driver);
+    }
+
+    @Test
+    void whenAUserWithTheNameSelectsSpectator_HisVoteShouldNotAppear() {
+
+        mainPage.openGlovoSite();
+        mainPage.joinRandomSession();
+        sessionPage.enterName(userName);
+        sessionPage.specClick();
+
+        Optional<WebElement> button2 = sessionPage.getVotingButton("2");
```

Comment:
Is there any other button here?
```suggestion
        Optional<WebElement> buttonWithPointValue2 = sessionPage.getVotingButton("2");
```